build/linux/installer/datafiles/base_container.data

```diff
@@ -134,43 +54,128 @@ MAINTAINER:              'Microsoft Corporation'
-/opt/microsoft/omsagent/plugin/health/health_model_buffer.rb;                                       source/plugins/ruby/health/health_model_buffer.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_model_builder.rb;                                      source/plugins/ruby/health/health_model_builder.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_model_constants.rb;                                    source/plugins/ruby/health/health_model_constants.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/parent_monitor_provider.rb;                                   source/plugins/ruby/health/parent_monitor_provider.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_model_definition_parser.rb;                            source/plugins/ruby/health/health_model_definition_parser.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_monitor_helpers.rb;                                    source/plugins/ruby/health/health_monitor_helpers.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_monitor_optimizer.rb;                                  source/plugins/ruby/health/health_monitor_optimizer.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_monitor_provider.rb;                                   source/plugins/ruby/health/health_monitor_provider.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_monitor_record.rb;                                     source/plugins/ruby/health/health_monitor_record.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_monitor_state.rb;                                      source/plugins/ruby/health/health_monitor_state.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_monitor_telemetry.rb;                                  source/plugins/ruby/health/health_monitor_telemetry.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_monitor_utils.rb;                                      source/plugins/ruby/health/health_monitor_utils.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/health_signal_reducer.rb;                                     source/plugins/ruby/health/health_signal_reducer.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/monitor_factory.rb;                                           source/plugins/ruby/health/monitor_factory.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/monitor_set.rb;                                               source/plugins/ruby/health/monitor_set.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/unit_monitor.rb;                                              source/plugins/ruby/health/unit_monitor.rb; 644; root; root
+/etc/fluent/plugin/lib/application_insights/version.rb;								   source/plugins/ruby/lib/application_insights/version.rb; 644; root; root
```

Comment:
Now that we can gem install can we consume this as gem ?

Reply:
AI ruby SDK deprecated & archived and there are not accepting prs any more.  there is already an active PR related to the proxy which we would need to use gem approach and in future, if we need any bug fixes and we will not able to use gem approach so we should continue to use the approach what we are using.